docs/descriptors/html_htmlhint.md

```diff
@@ -42,7 +42,7 @@ Use htmlhint in your favorite IDE to catch errors before MegaLinter !
 |   <img src="https://github.com/oxsecurity/megalinter/raw/main/docs/assets/icons/atom.ico" alt="" height="32px" class="megalinter-icon"></a>   | [Atom](https://atom.io/)                             | [atom-htmlhint](https://github.com/htmlhint/atom-htmlhint)                                   |                                               [Visit Web Site](https://github.com/htmlhint/atom-htmlhint){target=_blank}                                               |
 | <img src="https://github.com/oxsecurity/megalinter/raw/main/docs/assets/icons/brackets.ico" alt="" height="32px" class="megalinter-icon"></a> | [Brackets](http://brackets.io/)                      | [brackets-htmlhint](https://github.com/htmlhint/brackets-htmlhint)                           |                                             [Visit Web Site](https://github.com/htmlhint/brackets-htmlhint){target=_blank}                                             |
 | <img src="https://github.com/oxsecurity/megalinter/raw/main/docs/assets/icons/sublime.ico" alt="" height="32px" class="megalinter-icon"></a>  | [Sublime Text](https://www.sublimetext.com/)         | [SublimeLinter-contrib-htmlhint](https://github.com/htmlhint/SublimeLinter-contrib-htmlhint) |                                      [Visit Web Site](https://github.com/htmlhint/SublimeLinter-contrib-htmlhint){target=_blank}                                       |
-|  <img src="https://github.com/oxsecurity/megalinter/raw/main/docs/assets/icons/vscode.ico" alt="" height="32px" class="megalinter-icon"></a>  | [Visual Studio Code](https://code.visualstudio.com/) | [vscode-htmlhint](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint)     | [![Install in VsCode](https://github.com/oxsecurity/megalinter/raw/main/docs/assets/images/btn_install_vscode.png)](vscode:extension/mkaufman.HTMLHint){target=_blank} |
+|  <img src="https://github.com/oxsecurity/megalinter/raw/main/docs/assets/icons/vscode.ico" alt="" height="32px" class="megalinter-icon"></a>  | [Visual Studio Code](https://code.visualstudio.com/) | [vscode-htmlhint](https://marketplace.visualstudio.com/items?itemName=HTMLHint.vscode-htmlhint)     | [![Install in VsCode](https://github.com/oxsecurity/megalinter/raw/main/docs/assets/images/btn_install_vscode.png)](vscode:extension/HTMLHint.vscode-htmlhint){target=_blank} |
```

Comment:
If you see line 2 of this file, you'll see it has been entirely generated from descriptors :)

So you need to also update the descriptor, else the MD will be overwritten at the next doc build ^^

https://github.com/oxsecurity/megalinter/blob/9dcc80a15b91ab7e6f09752fe290cb48bcfab202/megalinter/descriptors/html.megalinter-descriptor.yml#L76

Reply:
ah ok - thanks! so I should close this PR and make a new one then I guess